js/messages.js

```diff
@@ -1,114 +1,133 @@
 const MESSAGES_CSV = "../contributor_data.csv";
 const CREDITS_CSV = "../credits_data.csv";
+const TWITTER_ART_CREDITS_CSV = "../twitter_art_data.csv";
 const pathname = window.location.pathname;
 const CURRENT_PAGE = pathname.slice(pathname.lastIndexOf("/"), pathname.length);
 
-function getCSVData(page, callback = undefined) {
-    let filename;
-    switch (page) {
-        case "/contributor-board.html":
-            filename = MESSAGES_CSV;
-            break;
-        case "/credits.html":
-            filename = CREDITS_CSV;
-            break;
-    }
+function getCSVData(filename, callback = undefined) {
+  Papa.parse(filename, {
+    download: true,
+    //To treat the first row as column titles
+    header: true,
+    complete: (result) => {
+      if (callback) {
+        callback(result.data);
+      }
+    },
+  });
+}
```

Comment:
main change: now that the credits section needs to load 2 csv's, I generalized this function to take filename instead of page